shelley-ma/impl/src/Cardano/Ledger/Allegra/Translation.hs

```diff
@@ -203,13 +253,14 @@ instance Crypto c => TranslateEra (AllegraEra c) PPUPState where
         }
 
 instance Crypto c => TranslateEra (AllegraEra c) TxId where
-  translateEra _ = error "TODO Shelley to Allegra translation"
+  translateEra _ = pure . coerce
```

Comment:
There is a default instance using `Coercible` (see the docstring of `TranslateEra`), I think you could use that.

Reply:
Ah, yes, I saw that (and removed a bunch of these) but clearly missed this one.